packages/driver/src/dom/coordinates.ts

```diff
@@ -7,12 +7,26 @@ const getElementAtPointFromViewport = (doc, x, y) => {
   return $elements.elementFromPoint(doc, x, y)
 }
 
-const isAutIframe = (win) => {
+const isAUTFrame = (win) => {
```

Comment:
Renamed to match our [other usages](https://github.com/cypress-io/cypress/blob/2e5fbad2c050b81ce75575802578c307808d61b5/packages/proxy/lib/http/response-middleware.ts#L232).